samples/cxl_host_exerciser/cxl_he_cmd.h

```diff
@@ -152,6 +238,15 @@ class he_cmd : public test_command {
     return true;
   }
 
+  void he_start_test() {
+      // start test
+      he_ctl_.Start = 1;
+      host_exe_->write64(HE_CTL, he_ctl_.value);
+      he_ctl_.Start = 0;
+      host_exe_->write64(HE_CTL, he_ctl_.value);
```

Comment:
This seems odd. I think that the hardware would want to clear the bit once the test was started.  That way, software could check whether the hardware has started.  May want to raise this to the hw team.

Reply:
fixed